app/models/task.rb

```diff
@@ -11,13 +11,14 @@ class Task < ActiveRecord::Base
   validates :start_date, :expected_end_date, :name, :company_id, :project_id, presence: :true
   validate :check_start_date, :check_start_and_end_date
   scope :get_tasks, ->(user){where('(tasks.assignable_id in (?) AND tasks.assignable_type = ?) OR (tasks.assignable_id in (?) AND tasks.assignable_type = ?)',Employee.all.team_employees_projects_tasks(user).pluck(:id) , "Employee",user.employee_teams.pluck(:team_id),"Team")}
+  scope :get_employee_tasks, ->(user){where('(tasks.assignable_id in (?) AND tasks.assignable_type = ?) OR (tasks.assignable_id in (?) AND tasks.assignable_type = ?)',user.id ,"Employee",user.employee_teams.pluck(:team_id),"Team")}
 
   def check_start_and_end_date
     errors.add(:expected_end_date,'cannot be before start_time') if (expected_end_date.to_date <= start_date.to_date)
```

Comment:
`errors.add(:expected_end_date,'cannot be before start_time') if (expected_end_date.to_date < start_date.to_date)`